msbuild/Xamarin.MacDev.Tasks.Core/Tasks/CreateAssetPackManifestTaskBase.cs

```diff
@@ -134,7 +134,7 @@ public override bool Execute ()
 				}
 
 				// update AssetPackManifestTemplate.plist
-				resource.Add ("URL", new PString ("http://127.0.0.1" + Path.GetFullPath (dir)));
+				resource.Add ("URL", new PString ("http://127.0.0.1" + Uri.EscapeUriString (Path.GetFullPath (dir))));
```

Comment:
I use `EscapeUriString` because I don't actually want to escape the string as if it were data. It looks like the path is used directly. Escaping as data would change the slashes and other things.